pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "maturin"
 [project]
 name = "netifaces2"
 description = "Portable network interface information"
-version = "0.0.21"
+version = "0.1.0"
```

Comment:
I don't want to bump the version with a minor yet, since I have reports of some users on windows that some calls to the win api causes huge memory spikes.

Reply:
ok will change back